src/Chunk.cpp

```diff
@@ -139,7 +139,8 @@ cChunk::~cChunk()
 	{
 		if (!(*itr)->IsPlayer())
 		{
-			(*itr)->Destroy(false);
+			// Scheduling a normal destruction is neither possible (Since this chunk will be gone till the schedule occurs) nor necessary.
+			(*itr)->DestroyNoScheduling(false);  // No point in broadcasting in an unloading chunk. Chunks unload when no one is nearby.
```

Comment:
It would perform no action if the addressed chunk were not valid, ideally.


On 13 Feb 2016, at 18:49, "LogicParrot" <notifications@github.com> wrote:

> In src/Chunk.cpp:
> 
> > @@ -139,7 +139,8 @@ cChunk::~cChunk()
> >  	{
> >  		if (!(*itr)->IsPlayer())
> >  		{
> > -			(*itr)->Destroy(false);
> > +			// Scheduling a normal destruction is neither possible (Since this chunk will be gone till the schedule occurs) nor necessary.
> > +			(*itr)->DestroyNoScheduling(false);  // No point in broadcasting in an unloading chunk. Chunks unload when no one is nearby.
> Actually, how exactly would DoWithChunk help in this scenario?
> 
> —
> Reply to this email directly or view it on GitHub.
>